src/components/QuranReader/SidebarNavigation/JuzSelection.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+
+import useTranslation from 'next-translate/useTranslation';
+
+import styles from './SidebarNavigation.module.scss';
+
+import Link from 'src/components/dls/Link/Link';
+import { getJuzIds } from 'src/utils/juz';
+import { getJuzNavigationUrl } from 'src/utils/navigation';
+
+const JuzSelection = () => {
```

Comment:
I feel the code is almost the same as `PageSelection`. Maybe we should have a component called `SidebarNavigationSelection` that accepts `SelectionType` and that handles both cases in-case we want to change the logic we don't need to change in 2 places